README.md

```diff
@@ -291,19 +291,24 @@ meta_gamma(meta, 0:2)
 
 
 ## Genetic diversity
-Multiallelic data can currently only be dealt with by the GitHub version of rdiversity.
 1. Initialise data:
 ```{r}
 library()
 vcf_file <- system.file("extdata", "pinf_sc50.vcf.gz", package = "pinfsc50")
-vcf <- read.vcfR(vcf_file, verbose = FALSE)
+#read in twice: first for the column names then for the data
+tmp_vcf <- readLines(vcf_file)
+vcf_data <- read.table(vcf_file, stringsAsFactors = FALSE)
+# filter for the columns names
+tmp_vcf <- tmp_vcf[-(grep("#CHROM",tmp_vcf)+1):-(length(tmp_vcf))]
+vcf_names <- unlist(strsplit(tmp_vcf[length(tmp_vcf)],"\t"))
```

Comment:
If these two lines are just to get the header information out, why don't you just drop the first and have
```R
vcf_names <- unlist(strsplit(tmp_vcf[grep("#CHROM",tmp_vcf)],"\t"))
```
instead of the second.